widget/accordioncontainer_internal_test.go

```diff
@@ -10,17 +10,136 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestAccordionContainer_toggleForIndex(t *testing.T) {
+func TestAccordionContainer_Toggle(t *testing.T) {
```

Comment:
It still depends on the unexported type accordionContainerRenderer, not sure how to access the button's OnTapped without going through the renderer

Reply:
Okay, didn’t see that.
You could tap it like so:
```go
for _, o := range test.LaidOutObjects(c.Content()) {
	if b, ok := o.(fyne.Tappable); ok {
		test.Tap(b)
		break
	}
}
```
This would make the test movable to `widget_test`. But I think the better way would be to transform the test into an image based one.

Reply:
I don’t insist on further improvement here :).
For me the PR is fyne now.